fido/fido.py

```diff
@@ -670,11 +714,14 @@ def walk_tar(self, filename, fileobj):
         except tarfile.TarError:
                 sys.stderr.write("FIDO: Error: TarError {0}\n".format(filename))
         finally:
-            if tarstream != None: tarstream.close()
+            if tarstream:
+                tarstream.close()
```

Comment:
Cool, thanks. I'm also wrapping the inner `tarstream.extractfile` call with a `contextlib.closing` as it doesn't provide a context manager. Very nice...